src/components/search/SearchSize.tsx

```diff
@@ -49,7 +49,7 @@ export const SearchSize: FC<SearchSizeProps> = ({ page, size }) => {
                     MenuProps={
                         {
                             'data-test': 'size-menu',
-                        } as any
+                        } as unknown as MenuProps
```

Comment:
we should avoid double casting, especially unknown and following a Type. Can we cast it directly to MenuProps? is there any solution here for avoiding this double casting approach?

Reply:
changed to `Partial< MenuProps>`